t/09_qesapdeployment.t

```diff
@@ -462,12 +462,12 @@ subtest '[qesap_execute_conditional_retry] retry after fail with expected error
             return @results; });
```

Comment:
You can expand this one to also store logname for each call and then check numeration of them

```
my @calls;
my @logs;


$qesap->redefine(qesap_execute => sub {
            my (%args) = @_;
            push @calls, $args{cmd};
            push @logs, $args{logname};
            my @results = (pop @return_list, 0);
            return @results; });
 ```